tfjs-backend-webgpu/src/shader_preprocessor.ts

```diff
@@ -394,18 +397,18 @@ function getSetOutputSnippet(
 function getInputSamplingSnippet(
```

Comment:
->`getInputSnippet`? Maybe also change `SAMPLING_SNIPPETS` to a better name in this file. Same to `outputBufferStr`.